cmd/xenia/scrquery/comments_by_asset.json

```diff
@@ -6,10 +6,10 @@
       {
          "name":"comments_by_asset",
          "type":"pipeline",
-         "collection":"nyt_coral_assets",
+         "collection":"comments",
```

Comment:
Seems like the query is pulling comments from the comment collection based on similar asset_id's. I don't see this comments_by_asset in-use anywhere, but the other (similar) query, comments_by_user is shown in this search [Cay - Comments By User](https://github.com/coralproject/cay/search?utf8=%E2%9C%93&q=comments+by)

I don't believe the comments by asset query is implemented or in-use yet, so you're correct that the query could change. I'm just assuming that we're going to pull comments from the comments collection as shown below:

> db.comments.findOne()
{
	"_id" : ObjectId("5723a8910e206b0006a41190"),
	"user_id" : ObjectId("5723a8900e206b0006a40f3d"),
	"asset_id" : ObjectId("57239ed60e206b0006a38379"),
	"body" : "Love this column!",
	"status" : "2",
	"date_created" : ISODate("2016-03-15T19:54:24Z"),
	"date_updated" : ISODate("2016-03-15T19:55:09Z"),
	"date_approved" : ISODate("2016-03-15T19:55:06Z"),
	"source" : {
		"id" : "10673457",
		"user_id" : "72537188",
		"asset_id" : "3415286"
	}
}

Where we have comments associated with an asset_id and can query for them. This PR is just meant to keep xenia in-line with the work that Pillar does.